perf/conf/log4j.properties

```diff
@@ -10,7 +10,7 @@ log4j.appender.Console.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} (%F:%M) -
 
 # Appender for Tachyon-Perf
 log4j.appender.PERF_SLAVE_LOGGER=org.apache.log4j.RollingFileAppender
-log4j.appender.PERF_SLAVE_LOGGER.File=${tachyon.perf.home}/logs/slave.log
+log4j.appender.PERF_SLAVE_LOGGER.File=${tachyon.perf.home}/logs/slave${tachyon.perf.slave.id}.log
```

Comment:
Could the id be just a number and hardcode the hyphen here? It seems a little weird for the id to be like `-1`. I think it would also make it easier to be used for other purposes.